gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,6 +66,11 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
+      require "awesome_spawn"
+      AwesomeSpawn.run("rake evm:db:raise_failover_executed_event")
```

Comment:
test was: change method to class method and invoke from some controller - it did worked